server/config.yaml.dist

```diff
@@ -3,11 +3,28 @@
 
 # shown in the website title and on the front page
 name: szurubooru
-# full url to the homepage of this szurubooru site, with no trailing slash
-domain: # example: http://example.com
 # used to salt the users' password hashes and generate filenames for static content
 secret: change
 
+# set to the root web address for your instance
+# example values:
+# - `/` (default) is used when the domain is unknown
+# - `https://szuru.example.com/` if you know the specific domain
+#      and is required if you want email-based password reset
```

Comment:
Also required for Twitter cards to work, as they require absolute (not relative) content URLs in the meta tags. (https://twittercommunity.com/t/card-error-unable-to-render-or-no-image-read-this-first/62736)